doc/reference/modules/basic_mapping.xml

```diff
@@ -106,9 +106,8 @@
                      <area id="hm5" coords="6 55"/>
                    <area id="hm6" coords="7 55"/>
                    <area id="hm7" coords="8 55"/>
-                 </areaspec>   
-                 <programlisting>
-           <![CDATA[<hibernate-mapping
+                 </areaspec>
+                 <programlisting><![CDATA[<hibernate-mapping
```

Comment:
This is what wrecks numbering [here](http://nhibernate.info/doc/nhibernate-reference/mapping.html#mapping-declaration-mapping).